docs/web-sdk/passing-custom-ondevicechange-handler.md

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
This file should be deleted. It contains Web SDK details which we are not pushing right now (as mentioned earlier).